recipes/knit/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.2" %}
+{% set name = "knit" %}
+
+package:
+    name: {{name}}
+    version: {{version}}
+
+source:
+  fn: {{name}}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/k/{{name}}/{{name}}-{{ version }}.tar.gz
```

Comment:
Instead of `/k/` we do `{{ name[0] }}` as it is always the first letter of the package name. Not that there really is a need to change it here as the package name is fixed. Just noting it for reference.